src/conn/conn_api.c

```diff
@@ -1294,21 +1294,41 @@ __conn_calibrate_ticks(WT_SESSION_IMPL *session)
 {
 #if (defined __i386) || (defined __amd64)
 	struct timespec start, stop;
-	uint64_t diff_nsec, diff_tsc, tsc_start, tsc_stop;
+	uint64_t diff_nsec, diff_tsc, min_nsec, min_tsc;
+	uint64_t tries, tsc_start, tsc_stop;
 	volatile uint64_t i;
 
-	__wt_epoch(session, &start);
-	tsc_start = __wt_rdtsc(session);
+	min_nsec = min_tsc = 0;
```

Comment:
It would make me feel better if `min_nsec` was initialized to 1 - then we could never end up with a divide by 0 from this code.

Reply:
I came up with another potential cause for the original problem, or really a form of the first one: the system's clock has insufficient granularity to register nanoseconds. Hopefully increasing the loop by 100x alleviates that. I have mixed feelings on your suggestion because although it avoids the divide by zero we have no idea that the algorithm is not effective.  We'll just compute inaccurate values/time differences later. But having a divide by zero in the field is not a good thing.